.github/workflows/build.yml

```diff
@@ -46,3 +46,6 @@ jobs:
 
       - name: build and test
         run: mvn -B verify
```

Comment:
verify should run everything up to but not including install, so it might make sense to add the coveralls stuff to this. if you want to keep it separate, it might be better to remove the `clean test` from the command below, since it will already have been built and tested. also, wouldn't clean remove the build you're testing?